packages/ui/form/PhoneInput.tsx

```diff
@@ -18,10 +20,22 @@ function PhoneInput<FormValues>({
   onChange,
   ...rest
 }: PhoneInputProps<FormValues>) {
+  const [defaultCountry, setDefaultCountry] = useState("US");
```

Comment:
Make this a custom hook, `useDefaultCountryCode` - which should encapsulate the useEffect fetch logic.

In the pages/api call you should clearly specify how this will work for self-hosting 🙏 - e.g. on self hosting you return a `501 Not Implemented` error which you can then gracefully handle from the frontend.